src/NServiceBus.Core/Routing/SubscribeContext.cs

```diff
@@ -21,7 +20,4 @@ public SubscribeContext(IBehaviorContext parentContext, Type[] eventTypes, Conte
     }
 
     public Type[] EventTypes { get; }
-
-    [EditorBrowsable(EditorBrowsableState.Never)]
-    public Type EventType => throw new NotImplementedException();
```

Comment:
Was this a replaced 1 with multiple but never cleaned up? Do we still need to go through obsoletion?

Reply:
Yeah it is private